regen/opcode.pl

```diff
@@ -1309,3 +1267,46 @@ sub generate_opnames_h_opcode_enum {
 	print $on "\n#define MAXO ", scalar @ops, "\n";
 	print $on "#define OP_FREED MAXO\n";
 }
+
+sub generate_opnames_h_opcode_predicates {
+	# Emit OP_IS_* macros
+	print $on <<~'EO_OP_IS_COMMENT';
+
+	/* the OP_IS_* macros are optimized to a simple range check because
+	    all the member OPs are contiguous in regen/opcodes table.
+	    opcode.pl verifies the range contiguity, or generates an OR-equals
+	    expression */
+	EO_OP_IS_COMMENT
+
+	my %OP_IS_SOCKET;					 # /Fs/
+	my %OP_IS_FILETEST;					 # /F-/
```

Comment:
comment is indented more than the ones that follow it

Reply:
original code indented comments were aligned with tabs, untabified